src/compas/geometry/bestfit/bestfit_numpy.py

```diff
@@ -122,8 +122,10 @@ def f(c):
         Ri = dist(*c)
         return Ri - Ri.mean()
 
-    xm = mean(x)
-    ym = mean(y)
+    # The mean of x and y are very nearly 0 (1.0e-15), which reveals a numerical
+    # instability of the problem.  So, we choose our initial guess
+    # to be an epsilon bigger than that.
+    xm = ym = 0.00001
```

Comment:
i guess that makes a lot of sense indeed because of the transformation to the local coordinate system of the points

Reply:
I've just been playing with this value a bit.  1e-8 and smaller gives crazy results, except 0.  0 and anything greater than 1e-7 seems to be fine.  But I will leave it as it is.